.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.vscode
```

Comment:
I don't think ignoring editor files belongs in repository `.gitignore` files. Every developer can instead configure it in their global gitignore. Otherwise, you end up having to ignore vim's swap files or Emacs backup files et cetera for basically every editor.

Reply:
Oh, I didn't know about global `.gitignore`! Thanks for the tip!